src/main/java/org/apache/logging/log4j/ext/kusto/Constants.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package org.apache.logging.log4j.ext.kusto;
+
+public class Constants {

```

Comment:
I don't think we want to have 1 class to define all constants in the app.

Since the first 4 (the ones which begin "LOG4J2_ADX_") are all used only by KustoStrategy.java, I suggest they belong there. And the last is used by KustoClientInstance.java, so I suggest putting it there.

I generally prefer `public`, but just to point out that we can make them `protected` vs `public` depending on whether we only want the relevant tests to use them, or foresee them being used more widely.